internal/mq/mqimpl/rocksmq/client/client_impl_test.go

```diff
@@ -325,20 +330,23 @@ func TestRocksmq_Properties(t *testing.T) {
 	_, err = producer.Send(msg)
 	assert.NoError(t, err)
 
-	msg = &mqwrapper.ProducerMessage{
-		Payload:    msgb,
-		Properties: map[string]string{common.TraceIDKey: "b"},
-	}
-	_, err = producer.Send(msg)
-	assert.NoError(t, err)
-
 	msgChan := consumer.Chan()
 	msgConsume, ok := <-msgChan
 	assert.True(t, ok)
 	assert.Equal(t, len(msgConsume.Properties()), 1)
 	assert.Equal(t, msgConsume.Properties()[common.TraceIDKey], "a")
 	assert.NoError(t, err)
 
+	// rocksmq consumer needs produce to notify to receive msg
+	// if produce all in the begin, it will stuck if consume not that fast
+	// related with https://github.com/milvus-io/milvus/issues/27801
+	msg = &mqwrapper.ProducerMessage{
+		Payload:    msgb,
+		Properties: map[string]string{common.TraceIDKey: "b"},
+	}
+	_, err = producer.Send(msg)
+	assert.NoError(t, err)
+
```

Comment:
this shall only fix the ut problem. do we have any solution to fix the bug once for all?